indexer/services/roundtable/src/tasks/orderbook-instrumentation.ts

```diff
@@ -46,6 +47,15 @@ export default async function runTask(): Promise<void> {
       },
     );
     statOrderbook(crossedOrderbookLevels, market, 'crossed_orderbook');
+    const openOrders: string[] = await OpenOrdersCache.getOpenOrderIds(
+      market.clobPairId,
```

Comment:
does this need to be reverted?

Reply:
I think we can do a clean revert here since multiple commits were cherry-picked into the original PR and when I revert the indexer changes from main, I will keep this change